src/browserlib/extract-events.mjs

```diff
@@ -310,20 +310,37 @@ export default function (spec) {
     };
     // CSS Animations & Transitions uses dt/dd to describe events
     // and uses a ul in the dd to describe bubbling behavior
-    let bubbles;
+    let bubbles, iface;
     if (container.tagName === "DT") {
       const bubbleItem = [...container.nextElementSibling.querySelectorAll("li")]
         .find(li => li.textContent.startsWith("Bubbles:"));
       if (bubbleItem) {
         bubbles = !!bubbleItem.textContent.match(/yes/i);
       }
+      // CSS Animation & Transitions document the event in the heading
+      // of the section where the definitions are located
+      let currentEl = container.parentNode;
+      while(currentEl) {
+	if (currentEl.tagName.match(/^H[1-6]$/)) {
+	  break;
+	}
+	currentEl = currentEl.previousElementSibling;
+      }
+      const interfaceEl = currentEl.querySelector("code");
```

Comment:
```suggestion
      const interfaceEl = currentEl?.querySelector("code");
```

(We may have to process other specs that use `<dt>` and do not have a section heading at the same level (e.g. because definition table is wrapped in a `<div>` for some reason)